Tests/POEApi.Model.Tests/POEApi.Model.Tests.csproj

```diff
@@ -85,6 +85,8 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
+    <None Include="Resources\FragmentStash.json" />
```

Comment:
I don't think this file should be included, and Visual Studio isn't able to find it.